module/userProjects/validator/v1.js

```diff
@@ -33,6 +33,19 @@ module.exports = (req) => {
             req.checkBody("data").exists().withMessage("data is required");
             req.checkBody("data.transactionId").exists().withMessage("transactionId is required");
             req.checkBody("data.osid").exists().withMessage("osid is required");
+        },
+        listUserProjects:function(){
+            req.checkQuery('userInvolvement')
```

Comment:
what is this userInvolvment? @borkarsaish65

Reply:
I have removed the logic now only projects which are consumed will appear